### ISSUE TITLE
Sniffer css override idea

### ISSUE THREAD

**Issue by p4tient** [TurbineCSS/Turbine#118]
Example code:
    button
        padding: 5px 10px 6px;
        color: $colorLink;
        border: 2px solid white;
        box-shadow: 0 0 2px gray;

```
button // Only for IE
    browser: ie;
    overflow: hidden;
    width: auto;
```

Current version:
for non-IE browsers this code returns nothing

for IE becomes:

```
button
    padding: 5px 10px 6px;
    color: $colorLink;
    border: 2px solid white;
    box-shadow: 0 0 2px gray;
    overflow: hidden;
    width: auto;
```

I think would be better:
for non-IE browsers:

```
button
    padding: 5px 10px 6px;
    color: $colorLink;
    border: 2px solid white;
    box-shadow: 0 0 2px gray;
```

for IE:

```
button
    padding: 5px 10px 6px;
    color: $colorLink;
    border: 2px solid white;
    box-shadow: 0 0 2px gray;
    overflow: hidden;
    width: auto;
```

**Comment by SirPepe** [comment]
This would indeed be an improvement, but it won't be easy to implement. At the moment the parser first merges all properties of identical selectors and the plugin comes into action _after_ that. What we would need to address this is maybe a new plugin hook (`while_parsing` or something) that could enable the browser sniffer to take action before the properties are merged. A good idea, but no small change.

**Comment by SirPepe** [comment]
There is a `while_parsing` hook now in dev. One could now modify the sniffer in a way that it collects the properties and values before selectors are merged and eleminates them afterwards. Wouldn't be too hard I guess...

**Comment by p4tient** [comment]
Thanks. It will be so helpful.

**Comment by SirPepe** [comment]
Fixed in d60bc93d8e909ddadcf87b15b481d1edb4230beb, will be in 1.1.0
